dotCMS/src/integration-test/java/com/dotmarketing/cms/urlmap/URLMapAPIImplTest.java

```diff
@@ -668,4 +659,43 @@ private UrlMapContext getUrlMapContext(final User systemUser, final Host host, f
     private UrlMapContext getUrlMapContext(final User systemUser, final Host host, final String uri) {
         return getUrlMapContext(systemUser, host, uri, PageMode.PREVIEW_MODE);
     }
+
+    /**
+     * methodToTest {@link URLMapAPIImpl#processURLMap(UrlMapContext)}
+     * Given Scenario: Process a URL Map url when both the Content Type and Content exists but the field to Map is a
+     * {@link com.dotcms.contenttype.model.field.DataTypes#INTEGER}
+     * ExpectedResult: Should return a {@link URLMapInfo} wit the right content ans detail page
+     */
+    @Test
+    public void shouldReturnContentletWhenTheContentExistsAndUseAIntegerFIeld()
```

Comment:
Is there any test for the happy path (non-number)

Reply:
Yes